spec/rails_helper.rb

```diff
@@ -81,6 +81,7 @@
   config.include Devise::Test::ControllerHelpers, type: :controller
   config.include ActiveSupport::Testing::TimeHelpers
   config.include ActiveJob::TestHelper
+  config.include ViewComponent::TestHelpers, type: :component
```

Comment:
Huh, a sign of things to come I guess :)

Reply:
Yep - testing components ;) I actually am starting to really like this pattern - it does feel like lego